interop/xds/client/client.go

```diff
@@ -280,26 +377,32 @@ func makeOneRPC(c testpb.TestServiceClient, cfg *rpcConfig) (*peer.Peer, *rpcInf
 	return &p, &info, err
 }
 
-func sendRPCs(clients []testpb.TestServiceClient, cfgs []*rpcConfig, ticker *time.Ticker) {
+func sendRPCs(clients []testpb.TestServiceClient, ticker *time.Ticker) {
 	var i int
 	for range ticker.C {
-		go func(i int) {
```

Comment:
My understanding is that the server is being updated so that the RPCs will hang long enough for a count on in-flight RPCs (versus RPCs failed due to circuit breaking) to be made. The old goroutine loops through each type of RPC (empty or unary). Now the loop is first and the go call is inside the loop so that each RPC is on its own goroutine and won't stop the RPC of the other type from being called if it hangs. I will update the ticker though so the timing is correct.

Reply:
But why not just add the goroutine? Why do you need to remove the old goroutine?

Reply:
The goroutine needs to happen on a basis per the elements in `cfgs`. I didn't remove the old goroutine, just moved it inside the `for _, cfg := range cfgs {...` loop, since previously it was on a per channel basis. I also had to take the logic of building the `savedWatchers` out since that needed to stay being done on a per channel basis. Otherwise the goroutine after the change should be the same as the one before.

Reply:
If I didn't missing anything, this one is not fixed.

Reply:
That's true. I was just thinking there's no need for this "diff". And it feels safer the old way.

Reply:
> That's true. I was just thinking there's no need for this "diff". And it feels safer the old way.

Some kind of change here was necessary.  The old code used one goroutine to send RPCs for all configs (serially).  These RPCs were not expected to block unless there was a problem.  Now some RPCs are expected to block indefinitely, so it's important for each RPC to happen in its own goroutine.  This diff moves the `go` to directly around the RPC itself, where it could block, instead of around the whole ticker invocation.  This seems fine to me.  The diffs that aren't real diffs because of indent changing are unfortunate but not problematic.

Let's get the manual tests done ASAP so we can submit this!